OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -520,6 +520,10 @@ Action InitInputPanel(Widget panel)
 					{ "StopKey", "Stop" },
 					{ "ScatterKey", "Scatter" },
 					{ "StanceCycleKey", "Cycle Stance" },
+					{ "StanceHoldFireKey", "Hold fire" },
+					{ "StanceReturnFireKey", "Return fire" },
+					{ "StanceDefendKey", "Defend stance" },
```

Comment:
Probably easiest to make a new two-wide "unit stances" section with two rows for the four modes.